packaging/pip_requirements.txt

```diff
@@ -1,17 +1,21 @@
 # minimal
-Pillow==8.2.0; python_version == '3.6'
-Pillow==9.0.0; python_version >= '3.7'
+# Pillow==8.1.2; python_version >= '3.7'
+Pillow==8.1.2; python_version >= '3.7' and python_version < '3.12'
+Pillow==10.3.0; python_version >= '3.12'
+torch==1.13.0; python_version == '3.7'
+torchvision==0.14.0; python_version == '3.7'
```

Comment:
Perhaps we could add these conditional lines below together with the other torch and torchvision lines? The same dependency in different condition should be considered as neighboring lines whenever possible.

Reply:
I think so too, I moved these dependencies within the same block so that we can better manage them

Reply:
Hmm I see, the problem is that the dependencies also have a fixed predefined order so it is better to move all these lines at the previous torch location. They are still together and still under the `# backends` section this way too.

Reply:
> Hi @pevogam, at https://github.com/intra2net/guibot/pull/52#discussion_r1570321115, so should I move all the dependencies in the #minimal to the #backends or restore it as it was before my commit

Just the torch-related dependencies to where they were

Reply:
I moved all the torch dependencies back to the `# backends` (return them to where they were before the commit)

Reply:
Great, resolving this.